package.json

```diff
@@ -2,20 +2,18 @@
   "name": "opticks",
   "version": "3.3.3",
   "description": "FindHotel Toggle Flag JavaScript SDK",
-  "main": "lib/simple.js",
+  "main": "lib/simple.ts",
+  "module": "lib/simple.mjs",
+  "types": "lib/simple.d.ts",
```

Comment:
@jopdeklein Should we change the entry point seeing as `simple` is deprecated? This will allow us to 

```
import {toggle} from 'opticks'
```

instead of

```
import {toggle} from 'opticks/lib/optimizely'
```

Reply:
That sounds like a great plan! I guess both imports will work so it's a non-breaking change (it's OK anyway as we're going to bump to 4.0.0 but it will make migrating even easier?)

Reply:
Correct, the current import implementation will still work. I've updated the entry point 👍